corehq/apps/registration/utils.py

```diff
@@ -157,8 +157,9 @@ def send_domain_registration_email(recipient, domain_name, guid, full_name):
         "wiki_link": WIKI_LINK,
         'url_prefix': '' if settings.STATIC_CDN else 'http://' + DNS_name,
         "is_mobile_experience": (
-            toggles.MOBILE_SIGNUP_REDIRECT_AB_TEST_CONTROLLER.enabled(recipient, toggles.NAMESPACE_USER) and
-            toggles.MOBILE_SIGNUP_REDIRECT_AB_TEST.enabled(recipient, toggles.NAMESPACE_USER))
+            toggles.MOBILE_SIGNUP_REDIRECT_AB_TEST_CONTROLLER.enabled(recipient)
+            and toggles.MOBILE_SIGNUP_REDIRECT_AB_TEST.enabled(recipient)
```

Comment:
I included `toggles.NAMESPACE_USER` thinking the namespace was necessary for `PredictablyRandomToggle`s. Does it work without that?

Reply:
my experience is that it should, because there is only one namespace defined, but i'll stick back in.